cms/static/js/views/xblock_outline.js

```diff
@@ -112,6 +112,7 @@ function($, _, gettext, BaseView, ViewUtils, XBlockViewUtils, XBlockStringFieldE
                 includesChildren: this.shouldRenderChildren(),
                 hasExplicitStaffLock: this.model.get('has_explicit_staff_lock'),
                 staffOnlyMessage: this.model.get('staff_only_message'),
+                enableHideFromTOC: this.model.get('hide_from_toc'),
```

Comment:
why is this change here? is this something I missed in [this PR](https://github.com/openedx/edx-platform/pull/33952) or is this just needed for these changes?

Reply:
In the other PR there is the `enableHideFromTOCUI` property, which allows us to know if this functionality is activated in the platform. For this PR, it's necessary to know if the functionality is enabled in the specific section, to show the share button.